packages/react-native-bridge/ios/Gutenberg.swift

```diff
@@ -226,9 +231,16 @@ extension Gutenberg: RCTBridgeDelegate {
     }
 
     public func extraModules(for bridge: RCTBridge!) -> [RCTBridgeModule]! {
+        // Aztec
         let aztecManager = RCTAztecViewManager()
         aztecManager.attachmentDelegate = dataSource.aztecAttachmentDelegate()
-        let baseModules:[RCTBridgeModule] = [bridgeModule, aztecManager]
+        
+        // Sentry
+        let sentryModule = RNSentry()
+        sentryModule.gutenbergBridge = bridgeModule
+        sentryModule.dataSource = dataSource
```

Comment:
The Sentry native module is explicitly instantiated here because it requires access to the bridge and data source.